testProjectBase/build_base.gradle

```diff
@@ -29,10 +29,10 @@ dependencies {
   protobuf files("ext/")
   testProtobuf files("lib/protos-test.tar.gz")
 
-  compile protobufDep
-  testCompile 'junit:junit:4.12'
+  implementation protobufDep
```

Comment:
This change allows `ProtobufJavaPluginTest` suite to run under gradle 7, but issues relating to copy dirs are causing gradle 7 `:jar` task to fail (failure is in test itself, not connected to plugin).  It requires some refactoring to enable 7.0 testing for `ProtobufJavaPluginTest`

Reply:
But `ProtobufJavaPluginTest` doesn't include Gradle 7 now, and `ProtobufKotlinDslCopySpecTest` is independent. So this change doesn't seem to be necessary now, does it?

Reply:
It is not necessary, no.  It just means when someone does look at gradle 7, the tests compile, so thought it was better to make the fix preemptively as it's backwards compatible to gradle 5.6.
Should I revert it?